manifest.json

```diff
@@ -57,13 +57,7 @@
       "default": "en_US"
     }, {
       "name": "tz",
-      "ask": {
-        "en": "Choose a time zone",
-        "fr": "Choisissez un fuseau horaire"
-      },
-      "choices": ["Etc/GMT","Etc/GMT+0","Etc/GMT+1","Etc/GMT+10","Etc/GMT+11","Etc/GMT+12","Etc/GMT+2","Etc/GMT+3","Etc/GMT+4","Etc/GMT+5","Etc/GMT+6","Etc/GMT+7","Etc/GMT+8","Etc/GMT+9","Etc/GMT-0","Etc/GMT-1","Etc/GMT-10","Etc/GMT-11","Etc/GMT-12","Etc/GMT-13","Etc/GMT-14","Etc/GMT-2","Etc/GMT-3","Etc/GMT-4","Etc/GMT-5","Etc/GMT-6","Etc/GMT-7","Etc/GMT-8","Etc/GMT-9","Etc/GMT0","Etc/Greenwich","Etc/UCT","Etc/UTC"],
-      "default": "Etc/UTC",
-      "example": "Europe/Paris"
+      "default": "Etc/UTC"
```

Comment:
Uh that sounds weird ? Why remove all these ? Is it meant to be a hidden setting ?